tests/src/services/KafkaConnectorTests.scala

```diff
@@ -40,7 +40,7 @@ import scala.language.postfixOps
 @RunWith(classOf[JUnitRunner])
 class KafkaConnectorTests extends FlatSpec with Matchers with BeforeAndAfter {
     implicit val transid = TransactionId.testing
-    implicit val ec = ExecutionContextFactory.makeCustomThreadPoolExecutionContext(poolSize = 4)
+    implicit val ec = ExecutionContextFactory.makeCachedThreadPoolExecutionContext()
```

Comment:
I may have missed the justification for having a custom context here? Now the test uses the same type of `ExecutionContext` as the production code.